src/Guide/Config.hs

```diff
@@ -38,8 +38,13 @@ data Config = Config {
   _adminPassword :: Text,         -- ^ Password for the admin user
   _prerender     :: Bool,         -- ^ Whether to prerender all pages when
                                   --    the app is started
-  _discussLink   :: Maybe Url     -- ^ Link to a place to discuss the site.
+  _discussLink   :: Maybe Url,    -- ^ Link to a place to discuss the site.
                                   --    Will be placed in the header
+  _portMain      :: Int,          --   CORS policy: port for main site.
```

Comment:
This isn't just for the CORS policy, this is for the whole site.

Also, use `-- ^` instead of `--` so that the comment would appear in Haddock documentation.

```suggestion
  _portMain      :: Int,          -- ^ Port for the main site.
```